okrs_2019.md

```diff
@@ -0,0 +1,209 @@
+  - *  [ ] Call graph analysis for JS code
+  - * [ ] Call graph analysis for Go code
+- * [ ] Add time-based cred analysis
+  - * [ ] Able to display cred for specific time ranges (e.g. Jan 2019)
+  - * [ ] Able to apply overall weighting function based on when a contribution was made
+- * [ ] Add support for heuristics to SourceCred
+- * [ ] Add heuristics or plugins for detecting several forms of project stewardship
+- * [ ] Come up with robust metrics for measuring attribution quality
+- * [ ] Add 'cred decomposition'
+
+  (e.g. who has cred for a particular file, or for documentation work in general)
+
+### Metrics
+
+Coming up with metrics for attribution quality is challenging, since there
+isn't a readily available ground truth to compare against. Right now, the best
+metric we have is asking the contributors to a project to rate the quality of
+our attribution. The "cred rating" in the table below is the average rating
+that contributors to a project give its cred attribution, measured on a
+10-point scale.
+
+We also would like to include explicit heuristics for a number of the 'Project
+Stewardship' tasks described above.
+
+| Metric                                  | Goal | Q4 | Q1 | Q2 | Q3 | Q4 |
+| --------------------------------------- | ---- | -- | -- | -- | -- | -- |
+| SourceCred's cred rating                | 8/10 | -  |    |    |    |    |
+| IPFS's cred rating                      | 7/10 | -  |    |    |    |    |
+| # of measured Project Stewardship Tasks | 3    | 0  |    |    |    |    |
+| # of initiatives completed              | 6    | 0  |    |    |    |    |
+
+
+## SourceCred Community Growth
+
+We envision SourceCred being a thriving open-source community which is
+passionate about recognizing and rewarding open-source contributors, about the
+art of software engineering itself, and, of course, about building SourceCred.
+
+Right now we have a pretty good start in that the founders are passionate about
+these things ;) but we hope to grow the engaged community of contributors and
+users.
+
+On the initiative side, we want to improve the documentation around all parts of
+SourceCred so that the project is easier to get up to speed on. We also intend
+to produce content explaining SourceCred, like blog posts and video talks.
+
+### Initiatives
+- * [ ] Improve SourceCred's documentation
+  - * [ ] Document the overall vision, mission, and philosophy
+  - * [ ] Document the overall technical architecture
+  - * [ ] Document the attribution algorithm (e.g. by writing a whitepaper)
+  - * [ ] Document the plugin system, and how to add a new one
+  - * [ ] Add API docs
+- * [ ] Produce engaging blog posts
+- * [ ] Produce engaging audiovisual content
+
+### Metrics
+
+| Metric                                  | Goal | Q4 | Q1 | Q2 | Q3 | Q4 |
+| --------------------------------------- | ---- | -- | -- | -- | -- | -- |
+| # of contributors w/ >10 cred           | 15   | 2  |    |    |    |    |
```

Comment:
> \# of contributors w/ >10 cred

I suspect that this is infeasible. Git itself is by any means a
successful open source project, with regular ICs both as independents
and as employed by leading tech companies, and it has merely 11
contributors with ≥10 cred. (For comparison, Git has 479 contributors
each of whom has at least 5 commits, and has 74 contributors each of
whom has at least 74 commits.)

Reply:
Good point. I've removed it pending us coming up with something that we have confidence in.